File: Easy/stunt_performance/stunt_performance.py
import sys
input=sys.stdin.readline
n=int(input())
buil=list(map(int,input().split()))
cnt,ans=1,1
for fi,bi in zip(buil,buil[1:]):
	if (fi>bi):
		cnt+=1
		ans=max(ans,cnt)
	else:
		cnt=1
print(ans)

File: Easy/stunt_performance/stunt_performance.cpp
#include <iostream>
#include <algorithm>
using ll=long long;
using namespace std;
int main()
{
    ios::sync_with_stdio(false);
    cin.tie(nullptr);
    ll n; 
    cin>>n;
    ll buil[n];
    for (ll i=0;i<n;i++) 
        cin>>buil[i];
    ll cnt=1;
    ll ans=1;
    for (ll i=1;i<n;i++)
    {
        if (buil[i]<buil[i-1]）
    {
        cnt++;
        ans=max(ans,cnt);
    }
        else cnt=1;
    }
    cout<<ans;
    return 0;
}
